src/main/java/mathandel/backend/utils/UrlPaths.java

```diff
@@ -45,4 +53,25 @@
     // /api/users/me/password
     public static final String userMePasswordPath = userMePath + "/password";
 
+    // /api/moderatorRequests
+    public static final String moderatorRequestsPath = "/api/moderatorRequests";
```

Comment:
maybe we should change to moderator-requests and all paths accordingly

Reply:
yup